common/web/keyboard-processor/tests/dom/cases/domKeyboardLoader.spec.ts

```diff
@@ -19,8 +21,8 @@ const device: DeviceSpec = {
 
 describe('Keyboard loading in DOM', function() {
   afterEach(() => {
-    if (window[KeymanWeb]) {
-      (window[KeymanWeb] as KeyboardInterface).uninstall();
+    if (KeymanWeb) {
+      (KeymanWeb as KeyboardInterface).uninstall();
```

Comment:
```suggestion
      KeymanWeb.uninstall();
```

With the `declare let` on line 10 do we need the cast?